ARC/Arc.cs

```diff
@@ -93,26 +95,48 @@ private static async Task StartDiscordBot(IConfigurationRoot settings)
             LoggerFactory = logFactory
         };
 
-        _clientInstance = new DiscordClient(discordConfig);
+        _clientInstance = new DiscordClient(discordConfig)
+        {
+            ClientVersion = "2.0"
+        };
         _serviceProvider = ConfigureServices(settings);
 
         // Run any necessary steps before starting the bot here.
         ServiceProvider.GetRequiredService<UptimeService>();
         ServiceProvider.GetRequiredService<ModMailService>();
+        ServiceProvider.GetRequiredService<SlashCommandsService>();
+        ServiceProvider.GetRequiredService<InteractionService>();
+        ServiceProvider.GetRequiredService<BanAppealService>();
+        
 
         // Connect to discord!
         await _clientInstance.ConnectAsync();
         _clientInstance.Ready += ClientInstanceOnReady;
+        _clientInstance.ClientErrored += ClientInstanceOnClientErrored;
           
         await Task.Delay(-1);
     }
 
+    private static async Task ClientInstanceOnClientErrored(DiscordClient sender, ClientErrorEventArgs args)
+    {
+        var debug_log = await ClientInstance.GetChannelAsync(ulong.Parse(GlobalConfig.GetSection("discord:debug_log").Value));
+        var errorEmbed = new DiscordEmbedBuilder()
+            .WithTitle("Error!")
+            .WithColor(DiscordColor.Red)
+            .WithDescription($"***An error occured <t:{DateTimeOffset.Now.ToUnixTimeSeconds()}:R>!***\n```{args.Exception}```");
+
+        await debug_log.SendMessageAsync(errorEmbed);
+
+    }
+
     private static async Task ClientInstanceOnReady(DiscordClient sender, ReadyEventArgs e)
     {
         await Task.Run(() =>
         { 
             Log.Logger.Information($"Logged in as {sender.CurrentUser}");
             Log.Logger.Information($"Ready!");
+            //ClientInstance.BulkOverwriteGlobalApplicationCommandsAsync(new List<DiscordApplicationCommand>() { });
+            //ClientInstance.BulkOverwriteGuildApplicationCommandsAsync(975717691564376084, new List<DiscordApplicationCommand>() { });
```

Comment:
Probably not the best idea to leave uncommented code here.

FEATURE IDEA: Launch flag to clear slash commands